python/demos/demo_stokes_nest.py

```diff
@@ -97,8 +97,8 @@ def create_mesh_gmsh(
         gmsh.model.setPhysicalName(1, outlet_marker, "Fluid outlet")
 
         # Set number of threads used for mesh
-        gmsh.option.setNumber("Mesh.MaxNumThreads1D", MPI.COMM_WORLD.size)
-        gmsh.option.setNumber("Mesh.MaxNumThreads2D", MPI.COMM_WORLD.size)
+        # gmsh.option.setNumber("Mesh.MaxNumThreads1D", MPI.COMM_WORLD.size)
+        # gmsh.option.setNumber("Mesh.MaxNumThreads2D", MPI.COMM_WORLD.size)
```

Comment:
Shouldn't ruff pick these lines up for removal?
```suggestion
```

Reply:
I'll remove them, as the mesh stopped being deterministic with them.